schematic/models/validate_attribute.py

```diff
@@ -731,6 +769,31 @@ class ValidateAttribute(object):
     def __init__(self, dmge: DataModelGraphExplorer) -> None:
         self.dmge = dmge
 
+    def _login(
+        self,
+        access_token: Optional[str] = None,
+        project_scope: Optional[list[str]] = None,
+        columns: Optional[list] = None,
+        where_clauses: Optional[list] = None,
+    ):
+        # if the ValidateAttribute object already has a SynapseStorage object, just requery the fileview, if not then login
+        if hasattr(self, "synStore"):
```

Comment:
Under what situations that the if statement here:  `if hasattr(self, "synStore")` would get triggered?

Reply:
If the `ValidateAttribute` object already has a `synStore` attribute because it was used during the validation of another column